libraries/cms/application/site.php

```diff
@@ -563,79 +563,49 @@ protected function initialiseApp($options = array())
 		$user = JFactory::getUser();
 
 		// If the user is a guest we populate it with the guest user group.
+		// SMZ: why do we need this???
```

Comment:
Every variable has to be initialised at some point and in this case this is that point